### PR TITLE
[INLONG-5086][Manager] Support create Oracle databases and tables

### PR DESCRIPTION
**Pull request by haibo-duan**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- About #5086 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by haibo-duan** [comment]
@healchow @dockerzhang  pls review it.

**Review by kipshi — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/oracle/OracleJdbcUtils.java [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    /**
+     * Execute SQL command on Oracle.
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param sql SQL string to be executed
+     * @throws Exception on execute SQL error
+     */
+    public static void executeSql(Connection conn, String sql) throws Exception {
+        Statement stmt = conn.createStatement();
+        LOG.info("execute sql [{}] success !", sql);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/oracle/OracleJdbcUtils.java:83]: This successful log should be written after the execute method.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/oracle/OracleResourceOperator.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        OracleTableInfo tableInfo = OracleSinkDTO.getTableInfo(oracleSink, columnList);
+        String url = oracleSink.getJdbcUrl();
+        String user = oracleSink.getUsername();
+        String password = oracleSink.getPassword();
+        String tableName = tableInfo.getTableName();
+        Connection conn = null;
+        try {
+            conn = OracleJdbcUtils.getConnection(url, user, password);
+
+            //In Oracle, there is no need to consider whether the database exists
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/oracle/OracleResourceOperator.java:101]: Please add one blank after `//`, thanks.

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Pull request merged**